Migrations/migrations.sql

```
@@ -0,0 +1,243 @@
+        [id] [int] IDENTITY(1,1) NOT NULL,
+        [order_id] [int] NULL,
+        [item_id] [int] NULL,
+        [quantity] [int] NULL,
+        PRIMARY KEY CLUSTERED ([id] ASC)
+    )
+END
+GO
+
+-- Add FK and CHECK constraints to order_item
+IF NOT EXISTS (
+    SELECT * FROM sys.foreign_keys WHERE name = 'FK_order_item_menu'
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)